lib/centurion/deploy_dsl.rb

```diff
@@ -85,23 +69,38 @@ def health_check(method)
    set(:health_check, method)
   end
 
+  def defined_service
+    service = fetch(:service, {})
```

Comment:
@kremso, I added a PR on your fork against your branch so you can review it. If you merge it, it should show up here.